pkg/cmd/codespace/ssh.go

```diff
@@ -238,6 +369,37 @@ func (a *App) Copy(ctx context.Context, args []string, opts cpOptions) error {
 	return a.SSH(ctx, nil, opts.sshOptions)
 }
 
+type configOptions struct {
+	codespace  string
+}
+
+func newConfigCmd(app *App) *cobra.Command {
+	var opts configOptions
+
+	configCmd := &cobra.Command{
+		Use:   "config [-c codespace]",
+		Short: "Write OpenSSH configuration to stdout",
+		Long: heredoc.Docf(`
+			The config command generates ssh connection configuration in OpenSSH format.
```

Comment:
Can you give some insight here on why you might want to use this command?

Reply:
I [expanded on it](https://github.com/cli/cli/pull/4915/commits/71c9fa11f06034ceaa9dce3971f6942e6c94ebb2) a bit - how does this strike you?

Reply:
Much better!